tools/xdp/s2n-quic-xdp/src/task/testing.rs

```diff
@@ -0,0 +1,24 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use core::time::Duration;
+use rand::prelude::*;
+use tokio::time;
+
+pub async fn random_delay() {
+    let delay = thread_rng().gen_range(0..100);
+    if delay > 0 {
+        let delay = Duration::from_micros(delay);
+        trace!("sleeping for {delay:?}");
+        time::sleep(delay).await;
+    }
+}
+
+/// The number of items to send through the test queues
+pub const TEST_ITEMS: usize = 10_000;
```

Comment:
Before pushing I set this to `1_000_000` and let it run for about 30 minutes so I think we can be pretty confident in it holding the invariants defined in the test harness.